include/mdw_dslash5_tensor_core.cuh

```diff
@@ -475,6 +492,74 @@ namespace quda
     }
   }
 
+#else
+
+  template <int BlockDimX, int Ls, int M, int N, int M_PAD, int N_PAD, bool reload, class T>
+  __device__ inline void mma_sync_gemm(T op_a[], half *sm_a, half *sm_b, half *sm_c, const mma::WarpRegisterMapping &wrm)
+  {
+
+#ifdef USE_FP16_HMMA_ACCUMULATE
```

Comment:
Where does this get defined? Is this a remnant of devel/profiling, can we get rid of it?

Reply:
This macro is defined in `include/mma_tensor_op/hmma_m16n16k16_sm70.cuh` and `include/mma_tensor_op/hmma_m16n8k8_sm80.cuh`. It determined whether we are using the fp16 or fp32 accumulate for the MMA instruction.